graylog2-server/src/test/java/org/graylog2/indexer/searches/SearchesTest.java

```diff
@@ -607,7 +607,7 @@ public void determineAffectedIndicesOnlyReturnsAliasesIfTooManyIndicesAreFound()
 
         final TimeRange absoluteRange = AbsoluteRange.create(now, now.plusDays(numberOfIndices + 1));
 
-        assertThat(searches.determineAffectedIndices(absoluteRange, null)).containsExactly(indexSet.getWriteIndexAlias());
+        assertThat(searches.determineAffectedIndices(absoluteRange, null)).containsExactly(indexSet.getIndexWildcard());
```

Comment:
Good practice is to use constant values for expectations, insead of building it using actual code.
This is great example, why ;)